extensions/http/jetty/src/main/java/org/eclipse/dataspaceconnector/extension/jetty/ConfigBaseUriService.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.extension.jetty;
+
+import org.eclipse.dataspaceconnector.spi.BaseUriService;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.system.configuration.Config;
+
+import static org.eclipse.dataspaceconnector.extension.jetty.JettyExtension.WEB_HTTP_URI_BASE;
+
+public class ConfigBaseUriService implements BaseUriService {
+    protected static final String DEFAULT_BASE_URI = "http://localhost:8181";
```

Comment:
How do we synchronize/align this with the port an API will use? The base URI may need to just be the domain